### DISCUSSION TITLE
`deno doc` - usage for `--category-docs` and `--default-symbol-map`

### DISCUSSION THREAD

**Discussion by laurence-myers** [denoland/deno#26734] [Q&A]
When you call `deno doc`, how do you use the options `--category-docs` and `--default-symbol-map`? The descriptions mean nothing to me. I could not find example uses. I tried using the options myself, but I didn't observe any change when generating HTML docs.

> --category-docs
> 
> Path to a JSON file keyed by category and an optional value of a markdown doc.

https://docs.deno.com/runtime/reference/cli/documentation_generator/#documentation-options-category-docs

> --default-symbol-map
>
> Uses the provided mapping of default name to wanted name for usage blocks.

https://docs.deno.com/runtime/reference/cli/documentation_generator/#documentation-options-default-symbol-map

**Comment by CarcajadaArtificial** [comment]
I have the same problem. Have you found any updates? It would be awesome to see a working example of this documentation functionality.
